pages/tidal tools/analyze_database.md

```diff
@@ -133,3 +141,26 @@ You need to install Docker in order to complete the database analysis. This is b
 
 ##  What about security?
 The entire analysis takes place locally on your machine. The only data that is captured and sent from the analysis are the results of the analysis and metadata. No application data, source code, files or the contents of any files on your machine are ever copied or sent anywhere.
+
+## Advanced Configuration
+
+If you are analyzing an Oracle databaase and need to only use features that are
```

Comment:
Let's rewrite it into something like the following:

**To comply with the terms of the Oracle Standard Edition (SE) license you should set the `analyze_workload` parameter to `false` in your database analysis configuration file. For example,**

WDYT?